sources/functions/rtorrent

```diff
@@ -6,7 +6,7 @@ export codename=$(lsb_release -cs)
 
 function whiptail_rtorrent () {
   if [[ -z $rtorrentver ]] && [[ -z $1 ]]; then
-    if [[ ! ${codename} =~ ("xenial"|"jessie") ]]; then
+    if [[ ! ${codename} == "xenial" ]]; then
```

Comment:
Feels like this logic could now be inverted. Also, I don't think thhere's a need to support .4 or .3 specifically for xenial users, knowing it will be gone in a year

Reply:
I don't mind leave the support for older versions in Xenial -- if the versions people want to use, then continuing to support them requires basically no changes